ballerina/types.bal

```diff
@@ -50,14 +64,14 @@ public annotation AutoIncrementConfig AutoIncrement on record field;
 
 # Defines the configuration to indicate the associations of an entity.
 #
-# + key - The foreign key
+# + keyColumns - The foreign key column
 # + reference - The primary key of the other table
 # + cascadeDelete - If it is true, the corresponding records in the other table
 #                   will be deleted when deleting a record from one table
 # + cascadeUpdate - If it is true, the corresponding records in the other table
 #                   will be updated when updating a record from one table
 public type RelationConfig record {|
-    string[] key;
+    string[] keyColumns;
     string[] reference = [];
```

Comment:
Shall we make these two fields optional fields? and infer them if it is not specified.

```suggestion
    string[] keyColumns?;
    string[] reference?;
```

Reply:
[2122a98](https://github.com/ballerina-platform/module-ballerina-persist/pull/11/commits/2122a98b0b088688e1d52b857bb174e574673687)